storage/time-series/influxdb/storer/app/influx/DataWriter.js

```diff
@@ -15,19 +15,22 @@ class DataWriter {
    * @param {String} url   Url to access influxdb
    * @param {String} token  A token with write permission in all orgs
    * @param {String} defaultBucket  Bucket Name for all data write
+   * @param {Boolean} writeAsString
```

Comment:
Quando comitei essas alterações dos tipos de dados, esqueci de colocar o comentário no parâmetro writeAsString. Pra ficar parecido com o readAsString, sugiro colocar: Indicates whether to write Influxdb data as a String

Reply:
Essa é branch do keycloak, quando a correção for efetuada na development será enviada para cá.